cartoframes/data/observatory/enrichment/enrichment_service.py

```diff
@@ -262,7 +262,7 @@ def _build_polygons_column_with_aggregation(variable, aggregation, column_sufix=
                 enrichment_table.{column} * (
                     ST_AREA(ST_INTERSECTION(enrichment_geo_table.geom, data_table.{geo_column}))
                     /
-                    ST_AREA(data_table.{geo_column})
+                    ST_AREA(enrichment_geo_table.geom)
```

Comment:
I've been wondering about the area in the denominator here. To avoid division by zero, maybe there should be a `nullif(st_area(...), 0)` to avoid that error?